### DISCUSSION TITLE
Perché pubblicare in formato shp?

### DISCUSSION THREAD

**Discussion by pigreco** [pcm-dpc/DPC-Aggregati-Strutturali-ITC-NordOvest#2] [Q&A]
Buongiorno e grazie per il repository.

Due domande legate la formato scelto:

1. perché la scelta del formato shapefile?
2. con QGIS è possibile visualizzare i vettori usando URI [0] del file shp, ma github ci avvisa che la cartella della [provincia di Torino](https://github.com/pcm-dpc/DPC-Aggregati-Strutturali-ITC-NordOvest/tree/master/NordOvest/Piemonte/Torino) supera il limite di 1000 file (Torino ha oltre 300 comuni x 4 file superiamo i 1000 file) quindi per molti comuni non posso usare questa strada, come ovviare? @pierluigicara 

[0] https://youtu.be/XknjhXRGmHs

**Comment by pierluigicara** [comment]
Grazie per l'apprezzamento del lavoro.
Il limite dei 1000 file è un limite dei repositry free di Github. Nel caso specifico, questa limitazione riguarda (in tutta Italia) solo i comuni della provincia di Torino e Cuneo. Si tratta dunque di solo 2 casi su 107 province totali.
Ma a parte questo, la versione desktop di Github (disponibile free) consente di clonare l'intero repository ovviando a tali limitazioni.
La scelta di usare il formato shape è legata al fatto che (come documentato) nella predisposizione dei dati è stato utilizzato il software ArcGIS, ma il formato shape è comunque di pubblico dominio ed è utilizzato da tutti gli altri software GIS (open source o no, desktop o server).
Abbiamo pensato all'utilizzo anche di altri formati, ma le limitazioni di occupazione di memoria per i repository free di Github avrebbero comportato la generazione di ulteriori numerosi repository con le conseguenti complessità di natura gestionale e, quindi, l'ipotesi per il momento è stata esclusa per il prevalere dei costi sui benefci.
